pages/utils/screenshot-area.tsx

```diff
@@ -28,6 +28,19 @@ export default function ScreenshotArea({
   children,
 }: ScreenshotAreaProps) {
   const [documentHeight, ref] = useContainerQuery(() => document.documentElement.scrollHeight);
+
+  const isExceedingHeightWarning =
+    documentHeight && documentHeight > MAX_PAGE_HEIGHT
+      ? `Warning: this page has height ${documentHeight}px, which is above the limit of ${MAX_PAGE_HEIGHT}px.
+      Taking a screenshot of this page may not include the full content. Remove ScreenshotArea if using this page for
+      integration tests only, or distribute test samples into separate pages to make screenshot tests work correctly.`
+      : '';
+
+  // Reporting the excessive height as a console error makes integration and screenshot tests for this page fail.
```

Comment:
1. Will this break any existing test?
2. Why not make the page completely error out instead of showing a warning, to minimize chances of devs ignoring the warning? We currently do this when the number of permutations exceeds a certain threshold: https://github.com/cloudscape-design/components/blob/main/pages/utils/permutations-view.tsx#L28

Reply:
I originally intended to fix the DRP permutations only, but then more pages failed due to introducing this warning - so I fixed them all.

I think there is no strong reason to throw an error because any page with this warning will fail in the github actions because we run a11y tests on all pages. I don't mind the approach though

Reply:
OK, I thought they would fail in the internal pipeline. If they fail in the PR checks, I still think failing during local development is even better, but I won't block the PR over this.